### PR TITLE
Features/entities/GiveTutorial

### PR DESCRIPTION
**Pull request by zhaoan12**
Testing the push and pull request functions in git by adding a partially complete GiveTutorial entity which stores data for keeping track of the player's actions in the tutorial.

### CODE REVIEW

**Review by sbrad044 — APPROVED** [review]

**Review by siddharthgowda — DISMISSED** [review]

File: src/main/java/com/mg105/entities/GiveTutorial.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package com.mg105.entities;
+
+public class GiveTutorial {
+
+    private boolean Moved = false;
+    private boolean Attacked = false;
+    private boolean UsedItem = false;
```
Comment (6167656e74323431) [inline src/main/java/com/mg105/entities/GiveTutorial.java:7]: Sytle: Usually we use `camelCase` not `PascalCase` in java for class fields.

**Review by 6167656e74323431 — CHANGES_REQUESTED** [review]
Also missing javadoc.

**Review by siddharthgowda — APPROVED** [review]

**Review by 6167656e74323431 — APPROVED** [review]
LGTM

**Review by sbrad044 — APPROVED** [review]

**Comment by 6167656e74323431** [comment]
Merged during weekly meeting.

**Pull request merged**